sql/plan/create_role.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plan
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+// CreateRole represents the statement CREATE ROLE.
+type CreateRole struct {
+	IfNotExists bool
+	Roles       []UserName
+}
+
+// NewCreateRole returns a new CreateRole node.
+func NewCreateRole(ifNotExists bool, roles []UserName) *CreateRole {
+	return &CreateRole{
+		IfNotExists: ifNotExists,
+		Roles:       roles,
+	}
+}
+
+var _ sql.Node = (*CreateRole)(nil)
+
+// Schema implements the interface sql.Node.
+func (n *CreateRole) Schema() sql.Schema {
+	return nil
```

Comment:
All these DDL nodes should probably return OkResult rows

Reply:
Makes sense!